enterprise/metrics/src/test/java/org/neo4j/metrics/MetricsKernelExtensionFactoryIT.java

```diff
@@ -161,4 +163,27 @@ private void addNodes( int numberOfNodes )
             }
         }
     }
+
+    private long readMonotonicIncreasingLongValue( File metricFile ) throws IOException
+    {
+        try ( BufferedReader reader = new BufferedReader( new FileReader( metricFile ) ) )
+        {
+            String[] headers = reader.readLine().split( "," );
+            assertThat( headers.length, is( 2 ) );
+            assertThat( headers[0], is( "t" ) );
```

Comment:
can you please give some descriptive names to header indexes, that should help to figured out later on what was expected to be there with a bit of context from creator :)